selfdrive/car/gm/carcontroller.py

```diff
@@ -89,18 +92,18 @@ def update(self, CC, CS):
     time_and_headlights_step = 10
     tt = self.frame * DT_CTRL
 
-    if self.frame % time_and_headlights_step == 0:
+    if CS.CP.openpilotLongitudinalControl and (not CS.CP.radarOffCan) and (self.frame % time_and_headlights_step) == 0:
```

Comment:
Note: If radar _was_ visible, but OP wasn't doing long, we wouldn't want to be sending. Only when both are true would we want to be sending stuff on the obstacle bus (the ASCM case essentially)